Formula/open-mpi.rb

```diff
@@ -22,6 +22,7 @@ class OpenMpi < Formula
 
   option "with-mpi-thread-multiple", "Enable MPI_THREAD_MULTIPLE"
   option :cxx11
+  option "with-cxx-bindings", "Enable C++ MPI bindings"
```

Comment:
> Add --with-cxx-bindings option because MPI C++ bindings are
deprecated by the MPI-3.0 standard and are no longer installed by
default.

Never mind, ignore me! Can you add a `deprecated` to the option text?